app/build.gradle

```diff
@@ -108,6 +108,9 @@ dependencies {
     implementation "io.coil-kt:coil-svg:2.2.2"
     kapt 'com.github.bumptech.glide:compiler:4.14.2'
 
+    // Image Zooming Library Compose (Need better alternatives)
+    implementation 'com.github.SmartToolFactory:Compose-Zoom:0.5.0'
```

Comment:
I would rather want us to keep our own implementation of this feature until google themselves bring out a native implementation of the feature. I only like third-party libraries when they are a standard and a necessity for our use cases, created by us (the devs) or if there is no way other than the library.

For now, please remove this and only add relevant code to the repo.

Reply:
Don't worry ill definitely remove this once i have my own iteration ready. Consider this jugaad for now.

Reply:
Done